ClosedXML/Excel/ConditionalFormats/IXLConditionalFormat.cs

```diff
@@ -119,6 +119,7 @@ public interface IXLConditionalFormat
         Boolean Bottom { get;  }
         Boolean Percent { get; }
 
+        IXLConditionalFormat StopIfTrue(bool value);
```

Comment:
I think you should add another overload method:
```c#
IXLConditionalFormat StopIfTrue();
```
which obviously defaults to `true`